src/controllers/confirmation.controller.ts

```diff
@@ -119,4 +115,18 @@ const route = async (req: Request, res: Response, next: NextFunction): Promise<v
   }
 };
 
+const getOverdueFiling = (companyProfile: PTFCompanyProfile): string => {
+  let overdueFiling: string = "";
+
+  if (companyProfile.isAccountsOverdue && !companyProfile.isConfirmationStatementOverdue) {
```

Comment:
I think this is the most readable format. @markpit what do you think?

Reply:
looks good

Reply:
there is actually a trick I discovered.
const getOverdueFiling = ({isAccountsOverdue, isConfirmationStatementOverdue}): string =>

call the method as normal, you can then use the fields from companyProfile directly!

Reply:
Yes, that seems to work a treat. Updated.